### PR TITLE
DM-21737: Issue Call for Proposals for Community Brokers

### PR DESCRIPTION
**Pull request by ebellm**
(this is a temporary PR for @leannep to comment on)

### CODE REVIEW

File: sections/community_brokers.tex [hunk 1]
```diff
@@ -189,3 +199,6 @@ \subsection{Timeline}
 		\end{itemize}
 	\item \textbf{Full LSST operations} \textit{currently Q4 2022}
 \end{itemize}
+}
+
+\newtext{\textbf{updated timeline with estimates of DPs, commissioning,  LOY1 ...}}
```
Comment (leannep) [inline sections/community_brokers.tex]: We should update this just before issue with the latest ComCam / LSSTCam /SV  survey schedule. The current Data Preview schedule can be found at https://docs.google.com/presentation/d/1tRHdSyXBRp850zfJO5NssHS-fGfGMMngDEfAD9Azdk4/edit#slide=id.p1

File: sections/components.tex [hunk 2]
```diff
@@ -35,7 +35,7 @@ \subsubsection{Alert Production}\label{sec:AP}
 AP may also be able to process non-standard visits with longer or shorter exposure times provided that, e.g., the visit image can be successfully PSF-matched and differenced with a template image.
 Alert generation in crowded fields may produce more than the maximum of 10,000 alerts per visit required to be supported by DM; alert generation in this circumstance is still under study.
 }.
-Due to the need for Data Release Production-derived templates, Alert Production cannot run at full scale and full fidelity during commissioning nor the first year of operations.  LSST DM is currently investigating options for Alert Production in year one.
+Due to the need for Data Release Production-derived templates, Alert Production cannot run at full scale and full fidelity during commissioning nor the first year of operations.  LSST DM is currently investigating options for Alert Production in year one \newtext{\citedsp{DMTN-107}}.
```
Comment (leannep) [inline sections/components.tex]: \newtext{\citedsp{LSE-459, DMTN-107}}.  
(LSE-459 is the new policy doc in draft)

File: sections/data_rights.tex [hunk 3]
```diff
@@ -12,3 +12,4 @@ \section{Data Rights to Alert Stream Components}\label{sec:data_rights}
 Instead, the LSST alert stream will be delivered to a to-be-determined set of community brokers (Section \ref{sec:community_brokers}).
 Any institution worldwide may propose to host a broker.
 Brokers may share (or not share) the contents of alert packets with whomever they choose, although the broker selection process (\S \ref{sec:community_brokers}) will prioritize selection of brokers that will provide world-public access to their services.
+\newtext{The contents of the \DIAObject, \DIASource, \DIAForcedSource, and \SSObject tables held in the PPDB are also public and may be freely shared, although the ability to access and query the PPDBs held in LSST DACs is restricted to data rights holders.}
```
Comment (leannep) [inline sections/data_rights.tex]: LSST DACs  in the USA and Chile is restricted ......

File: sections/community_brokers.tex [hunk 4]
```diff
@@ -128,9 +136,10 @@ \subsection{Determining the Number of Community Brokers} \label{sec:numbrokers}
 	\item the achieved purity of the alert stream
 \end{itemize}
 
-The proposal solicitation will provide updated estimates based on pipeline performance on precursor surveys.
-Final capacity determination will not be possible until commissioning data is available from ComCam, currently early in 2021.
+\oldtext{The proposal solicitation will provide updated estimates based on pipeline performance on precursor surveys.}
+Final capacity determination will not be possible until commissioning data is available\oldtext{from ComCam, currently early in 2021}.
```
[the remark this answers is not in the document]
Comment (ebellm) [reply sections/community_brokers.tex]: As written above it's not specified to be either ComCam or LSSTCam, which I think is desirable ("from ComCam" onwards is proposed for deletion).

File: sections/components.tex (hunk 2 above)
Comment (ebellm) [reply sections/components.tex]: I put this in but just as text for now; will update once the document is available somewhere people can find it.

File: sections/data_rights.tex (hunk 3 above)
Comment (ebellm) [reply sections/data_rights.tex]: (as we discussed on Slack the current text seems to cover this.)

Comment (leannep) [inline sections/data_rights.tex]: Michael asks whether a Broker may access proprietary LSST data and include it in their distribution. I think here we need to add a line stating that, if if a Broker team has data rights, they can access data in a DR but may only distribute any value-added data products to data rights holders.

File: sections/community_brokers.tex [hunk 5]
```diff
@@ -27,19 +27,21 @@ \subsection{Requirements for Community Brokers}
...
 Additionally, we anticipate that community brokers will require some level of support from the LSST operations team.
-Accordingly, LSST will conduct an open proposal process in order to select brokers with sufficient technical capability to enable valuable and reliable scientific returns.
+Accordingly, LSST \oldtext{will}\newtext{is} conduct\newtext{ing} an open proposal process in order to select brokers with sufficient technical capability to enable valuable and reliable scientific returns.
 
 We believe the scientific community will be best served by a rich broker ecosystem offering diverse capabilities.
-Any institution worldwide may submit a Letter of Intent (\S \ref{sec:LOIs}), and if invited, a full proposal.
-Demonstration of appropriate technical and personnel resources to support the proposed goals will be the minimum requirement to be considered.
+Any institution worldwide \oldtext{may}\newtext{was allowed to} submit a Letter of Intent (\S \ref{sec:LOIs}), and if invited, a full proposal.
```
Comment (leannep) [inline sections/community_brokers.tex]: Any institution worldwide \oldtext{may}\newtext{was allowed to} submit a Letter of Intent (\S \ref{sec:LOIs}). The LSST SAC decided to invite all institutions who submitted a LOI to submit a full proposal.

File: sections/components.tex [hunk 6]
```diff
@@ -7,7 +7,7 @@ \subsection{Facilities}
 
 \subsubsection{Data Acquisition, Transfer, and Processing}
 
-LSST will survey the sky repeatedly using standard or alternative standard visits (currently baselined as $2\times15$\,second exposures in a single band or a single 30\,second exposure, respectively).
+LSST will survey the sky repeatedly using standard or \newtext{``}alternative standard\newtext{''} visits (currently baselined as $2\times15$\,second exposures in a single band or a single 30\,second exposure, respectively).
 Immediately after the camera shutter closes, the images will be transferred via fiber networks to the LSST Data Facility (LDF; \S \ref{sec:LDF}) at the National Center for Supercomputing Applications (NCSA) in Illinois.
 All subsequent data processing leading to alerts occurs in the LDF.
```
[the remark this answers is not in the document]
Comment (ebellm) [reply sections/components.tex]: I realized we shouldn't say "shutter close" but "readout completes"--fixed here and elsewhere.

File: sections/data_rights.tex [hunk 7]
```diff
@@ -1,6 +1,6 @@
 \section{Data Rights to Alert Stream Components}\label{sec:data_rights}
 
-LSST data rights and data access policies are formally described in the LSST Data Rights Policy document, which is in development at this writing.
+LSST data rights and data access policies are formally described in the LSST Data Rights Policy document, \newtext{LDO-013}\oldtext{, which is in development at this writing}.
```
[the remark this answers is not in the document]
Comment (ebellm) [reply sections/data_rights.tex]: replaced with an actual citation.

**Pull request merged**